Misc/NEWS.d/next/C API/2021-10-21-15-00-43.bpo-39947.KOqrdL.rst

```diff
@@ -0,0 +1,3 @@
+Add new :c:func:`PyThreadState_SetProfile` and
+:c:func:`PyThreadState_SetTrace` functions to set the profile and the trace
```

Comment:
I would add "(replacing _PyEval_SetProfile)" and "(replacing _PyEval_SetTrace)" since those are likely to be the names people have if they are searching for changes, instead of just reading in advance.

Reply:
Also mention renaming and move is_tstate_valid to _PyThreadState_CheckConsistency

Reply:
is_tstate_valid() is a static function, it cannot be used outside ceval.c. We don't document changes which don't affected the public C API.

Reply:
> I would add "(replacing _PyEval_SetProfile)" and "(replacing _PyEval_SetTrace)" since those are likely to be the names people have if they are searching for changes, instead of just reading in advance.

Usually, we never mention private functions in the changelog.